src/java.base/share/classes/jdk/internal/misc/Signal.java

```diff
@@ -98,12 +98,12 @@ public String getName() {
      * @param other the object to compare with.
      * @return whether two <code>Signal</code> objects are equal.
      */
-    public boolean equals(Object other) {
-        if (this == other) {
+    public boolean equals(Object oth) {
```

Comment:
I'd  suggest to replace `oth` with `obj` and fix the `@param` clause in the method javadoc.